ssl4rs/data/transforms/boundary.py

```diff
@@ -41,11 +42,36 @@ def generate_boundary_mask_from_class_label_map(
         else:
             return dontcare_mask.astype(np.int64) * ignore_index
     roi_mask = roi_mask.astype(np.uint8) * 255  # to prep for opencv input
+
+    roi_mask = np.pad(roi_mask, pad_size, mode='constant', constant_values=0)
+
+
+    if roi_mask.sum().sum() == 0:
+        raise ValueError("no fields detected in roi mask")
```

Comment:
if this function is not DISA-specific, it shouldn't say "no fields", and instead return something empty, or at least have a more generic error message